app/src/main/java/org/oppia/app/player/state/itemviewmodel/SelectionContentViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.app.player.state.itemviewmodel
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+/** [ObservableViewModel] for MultipleChoiceInput values or ItemSelectionInput values. */
+@FragmentScope
+class SelectionContentViewModel @Inject constructor() : ViewModel() {
```

Comment:
Ditto for this interaction (please don't make it ``@Inject`` or scoped if it's going to be directly constructed).

Reply:
Done.